javalib/src/main/scala/java/util/Compat.scala

```diff
@@ -56,4 +56,32 @@ private[util] object Compat {
 
   }
 
+  // !!! Duplicate code with org.scalajs.testcommon.RPCCore.CollectionConvertersCompat
+  /** Magic to get cross-compiling access to `scala.jdk.CollectionConverters`
+   *  with a fallback on `scala.collection.JavaConverters`, without deprecation
+   *  warning in any Scala version.
+   */
+  object CollectionConvertersCompat {
```

Comment:
Maybe this name should contain JDK or Java somewhere so it is clear that this is about java collections?

Reply:
Done. Updated.